### ISSUE TITLE
bug: rke2 cis profile fix documentation

### ISSUE THREAD

**Issue by contributorr** [lablabs/ansible-role-rke2#269]
### Summary

Hello,

when using rke2 v1.30.5+rke2r1 and rke2_cis_profile: "cis-1.23", systemd rke2-server.service cannot start because of invalid profile. The issue started with rke2 1.30+ https://github.com/rancher/rke2/pull/5781

https://github.com/lablabs/ansible-role-rke2/blob/main/README.md?plain=1#L254

### Issue Type

Bug Report

### Ansible Version

```console
ansible [core 2.17.5]
  config file = /Users/user/.ansible.cfg
  configured module search path = ['/Users/user/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/homebrew/Cellar/ansible/10.5.0/libexec/lib/python3.12/site-packages/ansible
  ansible collection location = /Users/user/.ansible/collections:/usr/share/ansible/collections
  executable location = /opt/homebrew/bin/ansible
  python version = 3.12.7 (main, Oct  1 2024, 02:05:46) [Clang 16.0.0 (clang-1600.0.26.3)] (/opt/homebrew/Cellar/ansible/10.5.0/libexec/bin/python)
  jinja version = 3.1.4
  libyaml = True
```

### Steps to Reproduce

```yaml (paste below)
- name: Deploy RKE2
  hosts: all
  become: yes
  vars:
    rke2_cis_profile: "cis-1.23"
...
```

### Expected Results

Expected playbook to finish properly.

### Actual Results

```console
ANSIBLE
fatal: [hostname]: FAILED! => {"changed": false, "msg": "Unable to restart service rke2-server.service: Job for rke2-server.service failed because the control process exited with error code.\nSee \"systemctl status rke2-server.service\" and \"journalctl -xeu rke2-server.service\" for details.\n"}

SYSTEM LOG
Oct 25 22:43:02 hostname rke2[8451]: time="2024-10-25T22:43:02+02:00" level=fatal msg="cis-1.23 profile is deprecated. Please use 'cis' instead."
Oct 25 22:43:02 hostname systemd[1]: rke2-server.service: Main process exited, code=exited, status=1/FAILURE
```

**Comment by simonfelding** [comment]
set the `rke2_cis_profile: "cis"` variable to `"cis"` like it says in the error.

**Comment by MonolithProjects** [comment]
Docs fixed in #282
